_plugins/sort_edtf.rb

```diff
@@ -1,10 +1,46 @@
 module Jekyll
-    module SortEDTF
-        def sort_edtf(array_of_strings)
-            sorted = array_of_strings.map { |str| str.gsub('X', '0') }
-            sorted.sort_by { |str| str[/\d+/].to_i }
-        end
+  module SortEDTF
```

Comment:
@koilebeit Think about the scope of your functions. Make everything private that is not exposed (and should not be used outside of this module) see https://ruby-for-beginners.rubymonstas.org/advanced/private_methods.html